lib/Launcher.js

```diff
@@ -83,26 +83,32 @@ class Launcher {
 
     // Cleanup as processes exit.
     const listeners = [
-      helper.addEventListener(process, 'exit', killChromeAndCleanup),
-      helper.addEventListener(chromeProcess, 'exit', killChromeAndCleanup),
+      helper.addEventListener(process, 'exit', killChrome),
+      helper.addEventListener(chromeProcess, 'close', cleanupChrome),
     ];
     if (options.handleSIGINT !== false)
-      listeners.push(helper.addEventListener(process, 'SIGINT', killChromeAndCleanup));
+      listeners.push(helper.addEventListener(process, 'SIGINT', killChrome));
     try {
       const connectionDelay = options.slowMo || 0;
       const browserWSEndpoint = await waitForWSEndpoint(chromeProcess, options.timeout || 30 * 1000);
       const connection = await Connection.create(browserWSEndpoint, connectionDelay);
-      return new Browser(connection, !!options.ignoreHTTPSErrors, killChromeAndCleanup);
+      return new Browser(connection, !!options.ignoreHTTPSErrors, killChrome);
     } catch (e) {
-      killChromeAndCleanup();
+      killChrome();
       throw e;
     }
 
-    function killChromeAndCleanup() {
+    function cleanupChrome() {
       helper.removeEventListeners(listeners);
-      chromeProcess.kill('SIGKILL');
       removeSync(userDataDir);
     }
+
+    function killChrome() {
+      if (process.platform === 'win32')
+        childProcess.execSync(`taskkill /pid ${chromeProcess.pid} /T /F`);
```

Comment:
this is our heavy duty windows kill. Node reports `win32` for both 32 and 64 bit windows.